### ISSUE TITLE
Discussion forum and project 

### ISSUE THREAD

**Issue by jarlah** [guillaumepellegrino/acsrs#4]
Hi, 

Can you enable the discussions feature maybe? And add a project under Projects? im not able to make any project there :) But I don't need to if you make one.

The discussions feature will allow us to discuss things outside issues and PRs. The project will allow us to set milestones.

**Comment by guillaumepellegrino** [comment]
You are right, it should ease project discussions. It should be added, now :)
let me know if something is missing.
